### ISSUE TITLE
Chrome: NVDA cannot launch browse mode on a focused tab on startup

### ISSUE THREAD

**Issue by PratikP1** [nvaccess/nvda#7066]
Testing environment: Chrome Canary with latest updates
Windows insider build 15063 
NVDA version next-13979,c5f2944d

If Chrome is focused when NVDA is launched, NVDA is not in browse mode. Browse mode cannot be switched on or off with NVDA+spacebar. This behavior is limited to the tab which has focus. Switching to another tab shows that browse mode can be easily switched on or off. Returning to the original tab continues to display the behavior associated with that tab. See attached log. Most times, when NVDA is launched, it takes the user to the taskbar. This is inconsistent though. 

* Actual behavior: NVDA fails to launch browse mode.
* Expected behavior: Chrome tab with browse mode enabled.

STR:
1. Launch Chrome
2. Launch multiple tabs
3. restart NVDA. (This does not always result in NVDA focusing on Chrome.)
4. Try to switch to browse mode with NVDA+spacebar. 
5. switch to a different tab and observe browsing behavior.
6. Return to the original tab.

[NVDA on Chrome tab upon startup.txt](https://github.com/nvaccess/nvda/files/914090/NVDA.on.Chrome.tab.upon.startup.txt)

**Comment by derekriemer** [comment]
Hi:
This is a known issue. This occurs because there's no gainFocus on the window when NVDA restarts, therefore NVDA never realizes it should build a virtual buffer. A work around is to switch to something like the start menu, or the address bar, or a different app before rebooting NVDA.
Cheers.

**Comment by PratikP1** [comment]
While I agree that there is a work-around, let’s not expect every user to be aware of such a work around.

**Comment by jcsteh** [comment]
This is a duplicate of #5758. We're investigating a fix.
